Gemfile.lock

```diff
@@ -269,7 +269,7 @@ GEM
     rb-fsevent (0.11.0)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    rdkafka (0.8.1)
+    rdkafka (0.10.0)
```

Comment:
* Fixes for Ruby 3.0
* Allow any callable object for callbacks (gremerritt)
* Reduce memory allocations in Rdkafka::Producer#produce (jturkel)
* Use queue as log callback to avoid unsafe calls from trap context (breunigs)
* Allow passing in topic configuration on create_topic (dezka)
* Add each_batch method to consumer (mgrosso)
* Upgrade librdkafka to 1.5.0
* Add error callback config